modules/keycloak/ecs.tf

```diff
@@ -14,17 +14,18 @@ data "template_file" "app" {
   template = file("modules/keycloak/templates/keycloak.json.tpl")
 
   vars = {
-    app_image           = "nationalarchives/tdr-auth-server:${var.environment}"
-    app_port            = local.app_port
-    app_environment     = var.environment
-    aws_region          = var.region
-    url_path            = aws_ssm_parameter.database_url.name
-    username_path       = aws_ssm_parameter.database_username.name
-    password_path       = aws_ssm_parameter.database_password.name
-    admin_user_path     = aws_ssm_parameter.keycloak_admin_user.name
-    admin_password_path = aws_ssm_parameter.keycloak_admin_password.name
-    client_secret_path  = aws_ssm_parameter.keycloak_client_secret.name
-    frontend_url        = var.frontend_url
+    app_image                         = "nationalarchives/tdr-auth-server:${var.environment}"
+    app_port                          = local.app_port
+    app_environment                   = var.environment
+    aws_region                        = var.region
+    url_path                          = aws_ssm_parameter.database_url.name
+    username_path                     = aws_ssm_parameter.database_username.name
+    password_path                     = aws_ssm_parameter.database_password.name
+    admin_user_path                   = aws_ssm_parameter.keycloak_admin_user.name
+    admin_password_path               = aws_ssm_parameter.keycloak_admin_password.name
+    client_secret_path                = aws_ssm_parameter.keycloak_client_secret.name
```

Comment:
How much disruption would it cause to rename this parameter to distinguish it from your new client secret?

Reply:
Its going to need a lot of changes in a lot of places. I'm not sure its worth the effort.

Reply:
Ah, OK. It might be worth doing it if we find it causes confusion, but let's leave it for now.

Reply:
I've actually implemented the changes in this PR

Reply:
Sorry, I misunderstood! Thanks for updating the name. 👍